### PR TITLE
resurrect sidebar for some rogue nav pages

### PR DESCRIPTION
**Pull request by cpreid**
Resurrecting nav for some doc pages that dont live in nav. 
These pages still show up in search results, are linked to from other docs, and are indexed by search engines.

**This was a quick fix, and probably a safe bet between the two options**
- Add the nav to the rogue docs just to prevent the jarring experience. The links to them will work, and theyll remain in search results ✅
- Remove the pages entirely [requires finding any links that ref them and fixing]

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://66b13016468a5a00730edb78--cozy-fox-0defec.netlify.app

File: docs/experiments-plus/stratified-sampling.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: Stratified Sampling
 sidebar_label: Stratified Sampling
 slug: /experiments-plus/stratified-sampling
+displayed_sidebar: cloud
```
Comment (tore-statsig) [inline docs/experiments-plus/stratified-sampling.md:5]: this already has the sidebar

https://docs.statsig.com/experiments-plus/stratified-sampling/

File: docs/statsig-warehouse-native/features/autotune.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 title: Autotune (Beta)
 slug: /statsig-warehouse-native/features/autotune
 sidebar_label: Autotune
+displayed_sidebar: cloud
```
Comment (tore-statsig) [inline docs/statsig-warehouse-native/features/autotune.md:5]: already has a sidebar

https://docs.statsig.com/statsig-warehouse-native/features/autotune

**Review by tore-statsig — APPROVED** [review]

File: docs/statsig-warehouse-native/features/autotune.md (hunk 2 above)
Comment (cpreid) [reply docs/statsig-warehouse-native/features/autotune.md:5]: removed it!

File: docs/experiments-plus/stratified-sampling.md (hunk 1 above)
Comment (cpreid) [reply docs/experiments-plus/stratified-sampling.md]: removed it!

**Pull request merged**